source/wp-content/themes/wporg-parent-2021/theme.json

```diff
@@ -505,7 +505,7 @@
 			"fontSizes": [
 				{
 					"name": "Extra Small",
-					"size": "12px",
+					"size": "11px",
```

Comment:
This text is specified as 11px

![Screen Shot 2022-08-03 at 12 08 01 PM](https://user-images.githubusercontent.com/1017872/182497206-bacfd01d-9807-4cc0-8a15-03291fac22d1.jpg)

Reply:
The `12px` came from the style guide - I think you can either leave this text as 12px, or set it to a custom size of 11px.